internal/secrets/example_test.go

```diff
@@ -0,0 +1,41 @@
+package secrets_test
```

Comment:
Can you add unit tests that cover the interpolation functionality? Some things to cover might be 0 - N secrets, expecting errors on unsupported secret stores, etc.

Reply:
fixed